client/Assets/Scripts/CustomInputManager.cs

```diff
@@ -124,6 +124,12 @@ protected override void Start()
         UIControlsWrapper.GetComponent<CanvasGroup>().alpha = 0;
     }
 
+    public void Setup()
+    {
+        _player = Utils.GetPlayer(SocketConnectionManager.Instance.playerId);
+        directionIndicator = _player.transform.Find("AimDirection").GetComponent<AimDirection>();

```

Comment:
We can simplify this by doing:
```suggestion
_player.GetComponentInChildren<AimDirection>();
```